readthedocsext/theme/templates/base.html

```diff
@@ -52,6 +52,19 @@
       <div class="ui very padded container">
         {% block notifications %}
           {% include "includes/utils/messages.html" %}
+
+          {% if user.is_impersonate %}
+            <div class="ui small basic fitted center aligned segment">
+              <i class="fad fa-warning violet icon"></i>
```

Comment:
We should use more prominent styles here. This seems to be copied from a notification that intentionally doesn't have strong styling so it doesn't distract users.

In the case of the impersonate link, I like that the block on the dashboard now is prominent -- especially when I forget that I'm impersonating a user or I'm on the wrong tab.

```suggestion
            <div class="ui small center aligned warning message">
              <i class="fad fa-warning icon"></i>
```

![image](https://github.com/readthedocs/ext-theme/assets/1140183/68dede9c-fade-4b8c-be15-d53d94ac2463)